app/assets/stylesheets/components/_expander.scss

```diff
@@ -46,7 +46,7 @@
 .app-c-expander__icon {
   display: none;
   position: absolute;
-  top: 0;
+  top: 3px;
```

Comment:
Whilst this looks correct at a regular size, if you increase the font size it starts to look misaligned.

<img width="318" alt="Screenshot 2024-09-25 at 10 41 02" src="https://github.com/user-attachments/assets/6c79fe01-16e8-49b4-bc29-e8c31df2b693">

(taken on Firefox with 'zoom text only' enabled)

I've tried setting the height to 100% rather than 40px, which seems to improve things:

<img width="337" alt="Screenshot 2024-09-25 at 10 40 51" src="https://github.com/user-attachments/assets/601ceb78-848b-4f56-bc2c-db92af72a6df">

Outline added just for emphasis. Obviously would need updating in the option select component as well. Also might want to play with the `top: 3px` as well - this feels like a bit of a magic number and I'd prefer to avoid that.

Reply:
Yep, there's a lot of improvements that could be made here and I mentioned a full refactor in the other PR for the same tweak in the gem, citing magic numbers as part of the problem.

The best solution is to use flexbox and avoid all the outdated absolute positioning entirely, but I've had mixed reactions to using flexbox in other work I've done here, which I don't really understand (flexbox has been stable for years). I'll explore a flexbox solution instead, but I do hope it won't be squashed in preference to an outdated approach.

Reply:
Sure, either flexbox or the one line change I suggested is fine.

Reply:
@andysellick I've replaced the outdated absolute positioning solution with flex-box, avoiding the need for most magic numbers. I've tested it across different browsers and different zoom levels (and the Firefox 'Zoom text only' setting) and it's robust. 

If you're happy with it, I'll roll out the same change to the [option selects in the other PR](https://github.com/alphagov/govuk_publishing_components/pull/4256) as they use the same layout rules and so can receive this same fix.

Reply:
Sure go ahead.